packages/notebook-extension/src/index.ts

```diff
@@ -119,6 +119,8 @@ namespace CommandIDs {
 
   export const runAllBelow = 'notebook:run-all-below';
 
+  export const runAllMarkdown = 'notebook:run-all-markdown';
```

Comment:
Maybe better to name it `render-all-markdown`/`renderAllMarkdown` for consistency with the label?